src/command_modules/azure-cli-interactive/azclishell/az_completer.py

```diff
@@ -242,7 +243,7 @@ def gen_dynamic_completions(self, text):
                                 pass  # other completion method used
 
         # if the user isn't logged in
-        except (CLIError, AttributeError, ValueError):  # service client throws
+        except (CLIError, AttributeError, ValueError, ValidationError):  # service client throws
```

Comment:
Originally, I thought that it would be more clear to specify each, but I agree that it makes more sense to just catch all of them